lib/pwa.js

```diff
@@ -469,3 +475,12 @@ exports.updatePwaLighthouseInfo = function(pwa) {
 exports.count = function() {
   return db.count(ENTITY_NAME);
 };
+
+// Execute a list of Promise return functions in series
+exports.promiseSequential = function(list) {
```

Comment:
Yes, I was thinking that, I will create a utility file in lib and move there as well any other function that we may have in other files.